101-recovery-services-daily-backup-policy-create/azuredeploy.json

```diff
@@ -17,7 +17,7 @@
         "scheduleRunTimes": {
             "type": "array",
             "metadata": {
-                "description": "Times in day when backup should be triggered. e.g. 01:00, 13:00. This will be used in LTR too for daily, weekly, monthly and yearly backup."
+                "description": "Times in day when backup should be triggered. e.g. 01:00 or 13:00. Must be an array, however for IaaS VMs only one value is valid. This will be used in LTR too for daily, weekly, monthly and yearly backup."
```

Comment:
Assume this was tripping folks up - can we add a valid defaultValue for the example?

Reply:
good idea, will take care of that

Reply:
ok, can you add to this PR?